include/NovelRT/Exceptions/NullPointerException.h

```diff
@@ -0,0 +1,18 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_NULL_POINTER_EXCEPTION_EXCEPTION_H
+#define NOVELRT_NULL_POINTER_EXCEPTION_EXCEPTION_H
+
+#include <stdexcept>
+#include <string>
+
+namespace NovelRT::Exceptions {
+  class NullPointerException final : public std::runtime_error {
+  public:
+    NullPointerException(const std::string &message) :
```

Comment:
The reference symbols should be next to the type, not the parameter name. Also could we not standardise the message for this exception somehow?